src/components/RenderReview.jsx

```diff
@@ -0,0 +1,24 @@
+function RenderReview({review}) {
+
+
+    if (review.category) {
+        let lowerCaseGameCategory = review.category;
+        let UpperCaseGameCategory = lowerCaseGameCategory.charAt(0).toUpperCase() + lowerCaseGameCategory.slice(1);
+        
+        return (
+            <>
+            <h3>{review.title}</h3>
+            <img src={review.review_img_url} alt="Relevant picture of the game being reviewed"/>
```

Comment:
This isn't a great alt property would suggest using the title or something similar that is more relevant to the review.